src/gooso.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  $effect,
+  $familiar,
+  $item,
+  $items,
+  $location,
+  $monster,
+  $skill,
+  getBanishedMonsters,
+  have,
+  getModifier
+} from "libram";
+import { haveEffect, retrieveItem, use } from "kolmafia";
+import { ChronerQuest, ChronerStrategy, ChronerTask } from "./engine";
+import Macro from "./macro";
+import { chooseQuestOutfit, ifHave } from "./outfit";
+
+const location = $location`Moonshiners' Woods`;
+const banishers = $items`ice house, human musk`;
+
+export const gooso: ChronerQuest = {
+  name: "Gooso",
+  location,
+  tasks: [
+    ...banishers.map(
+      (i): ChronerTask => {
+        return {
+          name: `Banisher ${i}`,
+          completed: () => have(i) || getBanishedMonsters().has(i),
+          do: () => retrieveItem(i),
+          sobriety: "sober",
+        };
+      }
+    ),
+    {
+      name: "Blue Taffy",
+      completed: () => haveEffect($effect`Blue Swayed`) < 50,
+      do: () => {
+        retrieveItem($item`pulled blue taffy`);
+        use($item`pulled blue taffy`);
+      },
+      sobriety: "either",
+    },
```

Comment:
We need a price check on this, and we probably want this to only be sober because you can't use the skill when overdrunk